pkg/internal/checkup/checkup.go

```diff
@@ -402,6 +435,10 @@ func newTrafficGeneratorPod(checkupConfig config.Config, secondaryNetworkRequest
 		dstWestMACAddressParamName: checkupConfig.DPDKWestMacAddress.String(),
 		dstEastMACAddressParamName: checkupConfig.DPDKEastMacAddress.String(),
 		verboseParamName:           strings.ToUpper(strconv.FormatBool(checkupConfig.Verbose)),
+		PCIDeviceVarParamName:      pciDevicesVarName,
+	}
+	if checkupConfig.Verbose {
+		log.Printf("envVars: %v", envVars)
```

Comment:
Could you please split this change to a commit of its own?

Reply:
DONE